### ISSUE TITLE
save_model for yolov4-tiny_3l.cfg

### ISSUE THREAD

**Issue by ghost** [hunglc007/tensorflow-yolov4-tflite#264]
Hi,
Could you please help me to know what changes are to be made while converting the yolov4-tiny_3l.cfg in the save_model.py
In my understanding as the anchors in yolov4.cfg and yolov4-tiny_3l.cfg are same, but convolutional layers vary in yolo layers. Thus I assume that YOLOv4_tiny_3l is to be written accordingly. Could you please help me to write this function. Thank you.

**Comment by vinorth05** [comment]
I am also struggling with that for a moment now https://github.com/hunglc007/tensorflow-yolov4-tflite/issues/170
@hunglc007

**Comment by vinorth-v** [comment]
Hi, did you manage to solve it?

**Comment by ghost** [comment]
@vinorth05 unfortunately not. I am still looking for the solution. However, I am currently using yolov4-tiny.cfg.

**Comment by FUZhanhong** [comment]
@re-shubhamturai 
hi, dose yolov4-tiny work well?
I've changed tiny's anchors in config, but the result not good.
can you give me some adivce, thank you?

**Comment by ghost** [comment]
@FUZhanhong, yes, yolov4-tiny is working well. I presume there are no changes required in case of yolov4-tiny. Regarding changes in yolov4-tiny: I would suggest you could look into darknet repository issues. 
However, my issue is regarding yolov4-tiny_3l.cfg. I am sure @hunglc007 can put some light on it.

**Comment by vinorth-v** [comment]
It works with these changes:
https://github.com/hunglc007/tensorflow-yolov4-tflite/issues/261
